http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtAsyncHttpClient.java

```diff
@@ -111,67 +115,10 @@ private AwsCrtAsyncHttpClient(DefaultBuilder builder, AttributeMap config) {
             this.maxConnectionsPerEndpoint = config.get(SdkHttpConfigurationOption.MAX_CONNECTIONS);
             this.monitoringOptions = revolveHttpMonitoringOptions(builder.connectionHealthConfiguration);
             this.maxConnectionIdleInMilliseconds = config.get(SdkHttpConfigurationOption.CONNECTION_MAX_IDLE_TIMEOUT).toMillis();
-            this.proxyOptions = buildProxyOptions(builder.proxyConfiguration);
+            this.proxyOptions = buildProxyOptions(builder.proxyConfiguration, tlsContext);
         }
     }
 
-    private HttpMonitoringOptions revolveHttpMonitoringOptions(ConnectionHealthConfiguration config) {
```

Comment:
No changes have been made to these methods. I just moved them to the new class.

Reply:
gotcha